components/contentCard.vue

```diff
@@ -0,0 +1,51 @@
+<template>
```

Comment:
The file name should be ContentCard.vue. See https://vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended